app/src/main/res/layout/widget_button.xml

```diff
@@ -0,0 +1,27 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Screenshots of widget and configuration screen please.

Reply:
Widget on a home screen:
![Widget on a home screen](https://user-images.githubusercontent.com/8548596/71752256-62e83a00-2e4c-11ea-91f1-4d914a8f72ef.png)

Widget configuration screen:
![Widget configuration screen](https://user-images.githubusercontent.com/8548596/71752220-45b36b80-2e4c-11ea-8b78-8a1069a17e1c.jpg)